### ISSUE TITLE
[google] This app is blocked

### ISSUE THREAD

**Issue by avatar-lavventura** [wkentaro/gshell#22]
I keep having following error when I ran: `gshell about`, is there any way to handle this barrier to get the token id?

```
This app is blocked

This app tried to access sensitive info in your Google Account. To keep your account safe, Google blocked this access.
```

**Comment by Yusics** [comment]
I have the same issue

**Comment by samuelazran** [comment]
Same issue here

**Comment by wkentaro** [comment]
Is this issue still happening? I am now able to see the result from `gshell about`.

**Comment by avatar-lavventura** [comment]
@wkentaro Yes still happening. Please see for solution: [How to handle “App is temporarily blocked from logging in with your Google account” barrier](https://stackoverflow.com/a/65507155/2402577)

**Comment by wkentaro** [comment]
It seems the proposed solution needs client id/secret and editing the original file of gdrive.
How do you think it can be applied to `gshell`?

**Comment by avatar-lavventura** [comment]
While gdrive returns a valid information with the created `token_v2.json`,  gshell returns unauthorized output.

```
❯ gshell about
Failed to get about: Get https://www.googleapis.com/drive/v3/about?alt=json&fields=maxImportSizes%2CmaxUploadSize%2CstorageQuota%2Cuser: oauth2: cannot fetch token: 401 Unauthorized
Response: {
  "error": "unauthorized_client",
  "error_description": "Unauthorized"
```

Maybe there could be a way that gshell may able to use `token_v2.json` that is created by the `gdrive`.

I am not sure how could `gshell` may handle `client id/secret `

**Comment by avatar-lavventura** [comment]
I have created `~/.gdrive/token_v2.json` file using the approach on gdrive. Afterwards when I run `gshell init` seems like its able to connect into my gdrive account.

**Comment by HettyPatel** [comment]
having the same issue
